### PR TITLE
feat: 明细表行头单元格支持拖拽

### LINKED ISSUE #1637: 🙏明细表不使用序号，拖拽行高
### 🖋 Description
明细表，不显示序号列（showSeriesNumber: false）情况下，可以设置列表第一列的每一行  可拖拽数据单元格行高吗？（不知道是否叙述清楚了）
现在是明细表如果不显示序号，就不能拖拽**数据单元格**的行高。
image

### PR DESCRIPTION
**Pull request by wjgogogo**
### 👀 PR includes

✨ Feature

- [x] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [x] Solve the issue and close #1637 

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| 有序号时 | 无序号时 |
| ------ | ----- |
|      |      |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/cell/table-series-cell.ts [hunk 1]
```diff
@@ -1,89 +1,13 @@
...
-  isFrozenTrailingRow as isFrozenTrailingRowUtil,
-} from '../facet/utils';
-import {
-  getOrCreateResizeAreaGroupById,
-  getResizeAreaAttrs,
-} from '../utils/interaction/resize';
+import { TableDataCell } from './table-data-cell';
 
-export class TableSeriesCell extends DataCell {
+export class TableSeriesCell extends TableDataCell {
```
Comment (wjgogogo) [inline packages/s2-core/src/cell/table-series-cell.ts:5]: 从继承关系上来说，TableSeriesCell应该继承TableDataCell, 可以避免很多与TableDataCell重复的逻辑

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**